www/docs/client/header.md

```diff
@@ -0,0 +1,48 @@
+---
+id: header
+title: Custom header
+sidebar_label: Create Custom Header
+slug: /header
+---
+
+The headers option can be customize in config when using `withTRPC` in nextjs or `createClient` in react.js.
+
```

Comment:
In my opinion, this feels like a duplication of content, we already supply an example of setting headers here: https://trpc.io/docs/react#3-add-trpc-providers. I think what we need is documentation on the `onSuccess` and how it can be used to dynamically set headers on a mutation's success.

Reply:
Yes, I think that makes sense. Examples for NextJS and ReactJS would be better.